heclib/heclib_f/src/Support/ibits.f

```diff
@@ -1,7 +1,8 @@
       INTEGER FUNCTION  IBITS(IW,IBIT,NBITS)
       IX=0
       DO 10 I=0,NBITS-1
-10    IX=IBSET(IX,I)
+10    CONTINUE
+      IX=IBSET(IX,I)
```

Comment:
@perrymanmd  This commit looks like a typo.  However,  any reason we need to keep the file ibits.f ?   I can't see any usages.

Reply:
@ktarbet One of the compilers (I think the gfortran compiler with the -werror flag) wouldn't pass old-style DO loops (those that use a statement label) if the labeled statement was executable. So I had to make the labeled statement to be a CONTINUE statement in several places, which is what happened here.

I didn't search for calls to IBITS, so it may be obsolete. I imagine it was used by applications like DSSMATH and DATCHK to set portions of the quality codes